docs/UserGuide.md

```diff
@@ -2,41 +2,200 @@
 
 ## Introduction
 
-{Give a product intro}
+Trippie is a command-line app to plan any of your upcoming trips. It is tailored to match the needs of student travellers. Whether you are planning for your summer holiday, or a leisure trip on your student exchange program, Trippie got you covered! 
+
+* [Quick Start](#quick-start)
+* [Features](#features)
+    * [Create new trip: `new trip`](#creating-new-trip-new-trip)
+    * [View help: `help`](#viewing-help-help)
+    * [Add a place: `add`](#adding-a-place-add)
+    * [List all places: `list /p`](#listing-all-places-list-p)
+    * [Search for place: `search`](#searching-for-place-search)
+    * [Delete place: `delete /p`](#deleting-place-delete-p)
+    * [Edit budget: `budget`](#editing-budget-budget)
```

Comment:
General notes: 
+ 'convert /to'
+ 'edit functions' (ForEx Rate, ForEx Abbreviation)